BuildCliComponents.sh

```diff
@@ -1,58 +0,0 @@
-#!/bin/bash
```

Comment:
Why delete this file? It might need to be updated, but we need an option for non-Windows platforms.